packages/react-noop-renderer/src/createReactNoop.js

```diff
@@ -606,14 +610,50 @@ function createReactNoop(reconciler: Function, useMutation: boolean) {
       console.log(...bufferedLog);
     },
 
-    simulateErrorInHostConfig(fn: () => void) {
+    simulateErrorInHostConfigDuringBeginPhase(fn: () => void) {
       failInBeginPhase = true;
       try {
         fn();
       } finally {
         failInBeginPhase = false;
       }
     },
+
+    simulateErrorInHostConfigDuringCompletePhase(fn: () => void) {
```

Comment:
This new method was necessary to write a test that mimicked RN's `Text`/`View` nesting validation.